### PR TITLE
Add git workflow slides

### PR DESCRIPTION
**Pull request by timofurrer**
- [x] Diagrams need to be updated

### CODE REVIEW

File: sections/extras.tex [hunk 1]
```diff
@@ -10,6 +10,21 @@ \section{Extras}
...
 
+\subsection{What to commit and what not to commit}
+\begin{frame}[fragile]
+  \slidetitle
+  It's best practice to not commit:
+  \begin{itemize}
+    \item Generated Binaries
+    \item Large files (Use \lstinline{git-lfs})
+    \item Your personal editor configuration
+    \item ...
```
Comment (randombenj) [inline sections/extras.tex]: Passwords and Secrets :)

File: sections/remotes.tex [hunk 2]
```diff
@@ -207,12 +207,59 @@ \subsection{Exercises}
   \end{exercise}
 \end{frame}
 
-\subsection{Branching models}
+\subsection{git Workflows / Branching Models}
```
Comment (randombenj) [inline sections/remotes.tex]: s/git/Git/ ?

File: sections/extras.tex (hunk 1 above)
Comment (fliiiix) [reply sections/extras.tex]: but I store all my ssh keys in a git repo :stuck_out_tongue:

File: sections/remotes.tex (hunk 2 above)
Comment (timofurrer) [reply sections/remotes.tex]: please use suggestions next time :)

**Comment by fliiiix** [comment]
thanks @kraeki & @timofurrer :tada:

**Pull request merged**